crmsh/corosync.py

```diff
@@ -542,10 +542,15 @@ def create_configuration(clustername="hacluster",
     if transport is not None:
         transport_tmpl = "    transport: {}\n".format(transport)
 
+    if bindnetaddr is None:
+        bindnetaddr_tmpl = ""
+    else:
+        bindnetaddr_tmpl = "bindnetaddr: %s" % bindnetaddr
```

Comment:
Could you add the indentation (like for transport above) as well? Just to make the file look a bit nicer when created :)